Client/mods/deathmatch/logic/CClientGame.cpp

```diff
@@ -3424,6 +3424,9 @@ void CClientGame::Event_OnIngame()
     g_pGame->ResetAlphaTransparencies();
     g_pGame->ResetModelTimes();
 
+    //reset cursor color
```

Comment:
Just to be consistent with the other code. Please, add a space after two slashes and start a comment with first letter in upper case. Here and in other places as well.